CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## Next release
 
+### Drop Support for Python 3.8
+
+Mypy no longer supports running with Python 3.8, which has reached end-of-life.
+For the time being, it is still possible to type check older code using Python 3.9+
+by passing `--python-version 3.8` as argument. This will be dropped in the first half of 2025!
```

Comment:
```suggestion
When running mypy with Python 3.9+, it is still possible to type check code
that needs to support Python 3.8 with the `--python-version 3.8` argument.
Support for this will be dropped in the first half of 2025!
```
nit: it's not really about the age of the code